purchases/build.gradle

```diff
@@ -14,7 +14,8 @@ dependencies {
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
     implementation 'androidx.annotation:annotation:1.1.0'
-    api "com.android.billingclient:billing:$billingVersion"
+    newBillingApi "com.android.billingclient:billing:$billingVersion"
```

Comment:
should these be `newBillingImplementation` and `oldBillingImplementation`?

Reply:
We want this to be `api` so consumers can access the BillingClient classes. To give an example, if someone is doing:

```
val skuDetails: SkuDetails = product.skuDetails
```

If we do implementation, they would have to add a direct dependency on BillingClient.

Reply:
ohhhhh right, thank you 👍